services/data/operators/mysql.py

```diff
@@ -1,3 +1,28 @@
+"""
+Module to house the MySQL Operator to interact with a MySQL database.
+"""
+import mysql.connector
+
 class MySQL:
+    def __init__(self):
+        self.db = mysql.connector.connect(
```

Comment:
Would this be better in a config file - Presume these aren't your actual credentials. :)

Reply:
Yes, when using real credentials they will be in the .env file which won't be pushed. Until we set up the mysql together, the tables and stuff, this is how it will be.